Documentation/wpf.vsconfig

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Component.CodeAnalysis.SDK",
+    "Microsoft.Component.MSBuild",
+    "Microsoft.Component.VC.Runtime.UCRTSDK",
+    "Microsoft.Net.Component.4.6.2.SDK",
```

Comment:
After recent changes from Rob and me, we don't need 4.6.2 SDK any more.

Reply:
Thanks! I’ll rebase on master again. This is still experimental - I couldn’t get builds to trigger while keeping the PR marked as WIP... 

wpf.vsconfig in this PR is far from experimental. It’s well tested at this point (it can use pruning of 4.6.2 SDk though). On a brand new VM + fresh VS install using wpf.vsconfig, a new clone of master builds cleanly.

Reply:
As an aside we need 4.7.2 targeting pack to build PBT.